jenkins-jobs/jobs/DocumentationJob.groovy

```diff
@@ -94,8 +117,12 @@ def setupBuildSteps(job, doWebPublish, doMavenPublish,  properties = null) {
                 }
             }
             if (doWebPublish) {
-                shell('rm -rf $docsLocation/cpp/ && mkdir -p $docsLocation/cpp/ && cp -r ./build/docs/doxygen/html/* $docsLocation/cpp/')
-                shell('rm -rf $docsLocation/java/ && mkdir -p $docsLocation/java/ && cp -r ./build/docs/javadoc/* $docsLocation/java/')
+                shell(docCopyCommand("cpp", "doxygen/html/*"))
+                shell(docCopyCommand("java", "javadoc/*"))
+                shell(shuffleboardDocCopyCommand("shuffleboard/api"))
+                shell(shuffleboardDocCopyCommand("shuffleboard/plugin/base"))
+                shell(shuffleboardDocCopyCommand("shuffleboard/plugin/networktables"))
+                shell(shuffleboardDocCopyCommand("shuffleboard/plugin/cameraserver"))
```

Comment:
Is there a way to make this not require configuring every plugin?  If we add a plugin in the future, we would need to remember to add it here too.

Reply:
This repo is full of those. So that's not something I'm super worried about. Unless you have an idea how to fix it.

Reply:
I'd be possible to just list all the dirs in `build/docs/shuffleboard/plugin` and map those to shell commands

Reply:
But if we add plugins that don't need a public API, such as game-specific widgets, then it would imply it's able to be depended on when it's not really a good idea to do so

Reply:
A bit late to the party, but we could just have a file in shuffleboard/plugin that has all the repos that need to have documentation copied over, and just enumerate through it.